packages/app/src/components/account/index.tsx

```diff
@@ -34,14 +28,31 @@ export const AccountButton: FC<{ config: TenderizeConfig }> = ({ config }) => {
 
   const supportedChainIds = Object.keys(config.chainUrlMapping).map((i) => parseInt(i, 10));
 
+  const networkAvatar = (chain: ChainId | undefined) => {
+    switch (chain) {
+      case ChainId.Arbitrum:
+        return <Avatar size="small" src="/arbitrum.svg" />;
+      case ChainId.ArbitrumRinkeby:
+        return <Avatar size="small" src="/arbitrum.svg" />;
+      case ChainId.Rinkeby:
+        return <Avatar size="small" src="/ethereum.png" />;
+      case ChainId.Mainnet:
+        return <Avatar size="small" src="/ethereum.png" />;
```

Comment:
nevermind, thought the text is removed

Reply:
No because you will either run in prod or either in testnet mode for the entire app.

Reply:
but that doesn't stop a user from switching to another network - it should still be highlighted

Reply:
What should be highlighted ?

Reply:
Suggestion: 

Set supported networks based on environment (prod vs staging) to mainnets vs testnets so that in prod you can't switch to testnet and vice versa

Reply:
but what should happen if they switch in metamask? disconnect? what do we do when they are trying to connect? display that first they should switch network and then try to connect? This is why I don't understand the earlier change which included displaying the banner when you are not connected. We should let connecting despite being on any network, we are already handling it in the application.

Reply:
> This is why I don't understand the earlier change which included displaying the banner when you are not connected.

II explained on discord this is not intentional it is a side effect of now correctly initiating the wallet providers with which chains the app supports.

Reply:
> We should let connecting despite being on any network, we are already handling it in the application.

Or we could request to change the network when we receive the error